source/dealii/dealii_hierarchy_helpers.cc

```diff
@@ -205,8 +205,8 @@ DealIIHierarchyHelpers<dim, VectorType>::build_restrictor(
           dealii::Vector<ScalarType> interior_delta_eig(n_interior_elem);
           for (unsigned int k = 0; k < n_interior_elem; ++k)
           {
-            interior_delta_eig[k] = delta_eigenvector_matrix->operator()(
-                row, interior_dof_indices_map[k]);
+            interior_delta_eig[k] =
+                delta_eigenvector_matrix->el(row, interior_dof_indices_map[k]);
```

Comment:
I thought the whole point of this PR was to not use `el`

Reply:
The documentation says that `operator()()` is expensive and `el()` should be preferred in general. I only want to able to use the former when debugging. Hence, I added a second commit where I switch back to `el()`.